Add warnings of non existent periods (#29234)
mantidproject/mantid: docs/source/release/v5.1.0/muon.rst

- The ALC interface in workbench will now show errors by default. The error bars can also be turned on/off using the right-click plot menu.
- Have updated the FDA GUI so that it functions correctly for frequency transforms and single fits.
- Added in default group and pair selection when loading grouping files from xml.

> - Added warnings when not all selected runs have the relevant periods to calculate certain groups.

Reviewer: ```suggestion
- Added warnings when the selected runs do not have the relevant periods to calculate the groups or pairs.
```
